tests/openapi/test_responses.py

```diff
@@ -249,15 +248,15 @@ def test_additional_responses_overlap_with_other_responses() -> None:
     class OkResponse(BaseModel):
         pass
 
-    @get(responses={200: ResponseSpec(model=OkResponse)})
+    @get(responses={200: ResponseSpec(model=OkResponse, description="Overwritten response")})
     def handler() -> Person:
         return PersonFactory.build()
 
-    with pytest.raises(
-        ImproperlyConfiguredException,
-        match="Additional response for status code 200 already exists in success or error responses",
-    ):
-        create_responses(handler, raises_validation_error=True, generate_examples=False, plugins=[])
+    responses = create_responses(handler, raises_validation_error=True, generate_examples=False, plugins=[])
+
+    assert responses is not None
+    assert responses["200"] is not None
+    assert responses["200"].description == "Overwritten response"
```

Comment:
please add one more test- 

we need to test raises and then customizing the response

Reply:
be4dca8